corehq/apps/accounting/models.py

```diff
@@ -1431,10 +1431,6 @@ def send_ending_reminder_email(self):
             raise SubscriptionReminderError(
                 "This subscription has no end date."
             )
-        if self.is_renewed:
```

Comment:
you could replace this with an assert if you want the function to only run on non-renewed subscriptions